packages/ember-testing/lib/setup_for_testing.js

```diff
@@ -35,11 +35,13 @@ export default function setupForTesting() {
     setAdapter((typeof self.QUnit === 'undefined') ? new Adapter() : new QUnitAdapter());
   }
 
-  jQuery(document).off('ajaxSend', incrementPendingRequests);
-  jQuery(document).off('ajaxComplete', decrementPendingRequests);
+  if (jQuery) {
```

Comment:
@cibernox - Yes, it exports `undefined` in that case you can see ([from here](https://github.com/emberjs/ember.js/blob/master/packages/ember-views/lib/system/jquery.js#L3-L30)) that it basically does:

```js
let jQuery

export default jQuery
```